central/reports/scheduler/v2/reportgenerator/report_gen_impl.go

```diff
@@ -134,12 +134,6 @@ func (rg *reportGeneratorImpl) generateReportAndNotify(req *ReportRequest) error
 			zippedCSVData = nil
 			templateStr = noVulnsFoundEmailTemplate
 		}
-
-		messageText, err := formatMessage(req.DataStartTime, templateStr, req.ReportSnapshot.GetVulnReportFilters().GetImageTypes())
```

Comment:
I think we should do the formatting for default text here outside of `for _, notifierSnap := range req.ReportSnapshot.GetNotifiers` loop because the default text remains the same for all notifiers. In the loop we should just replace the messageText with customBody if it is set.